h2o-algos/src/main/java/hex/coxph/CoxPH.java

```diff
@@ -490,7 +492,7 @@ protected void calcModelStats(CoxPHModel model, final double[] newCoef, final Co
       }
     }
 
-    protected void calcCumhaz_0(CoxPHModel model, final CoxPHTask coxMR) {
+    protected void calcCumhaz_0(CoxPHModel model, DataInfo dinfo, final CoxPHTask coxMR) {
```

Comment:
the parameter doesn't seem to be used, is it?

Reply:
removed. Still works :-D